### ISSUE TITLE
Add environment files

### ISSUE THREAD

**Issue by Leticia-maria** [Leticia-maria/Introduction.jl#7]
- [ ] Install deps

**Comment by aritraroy24** [comment]
@Leticia-maria, do you mean `.env` file for julia by saying environment files? So that, we can store some confidential data?

**Comment by Leticia-maria** [comment]
No, this is for next versions of this course repo. In Julia, you can create environments and export/import packages globally, that is what I will do. People won't be installing packages locally.

**Comment by aritraroy24** [comment]
Ahh...Got it.
